t.js

```diff
@@ -0,0 +1,12 @@
+const Evervault = require('./');
+
+const evervault = new Evervault(
+  'NzYx:6UpD1v21T0ZuVJo9cbW2LhKA0Ug5GkVRrWMdJvdgn1PeLbUTIPhetBmxSBb0CpmAQ'
+);
+
+async function go() {
+  const encrypted = await evervault.encrypt('Adam Kelly');
+  console.log(encrypted);
```

Comment:
can we write a unit test that asserts the functionality works?

Reply:
Presume this wasn't meant to be committed

Reply:
yes, will remove and recommit